add documentation for AssetScanner (#119)
cognitedata/gearbox.js: src/components/AssetScanner/stories/full.md

#### Usage requirements:

Component depends on few third-part APIs, which is provided by Google
Vision (to recognize text on picture). The user has to insert props
to the component with a valid API key for Google Vision. Follow this
guide to generate an api-key:

**Google Vision API**

> Before you can use the Cloud Vision API, you must enable it for your project and generate your API key, see here how – [Enable the Vision API](https://cloud.google.com/vision/docs/before-you-begin).   

> Don't forget for setting up API key restrictions – [Using API Keys](https://cloud.google.com/docs/authentication/api-keys).  

Google Vision API key could be pass to component via **ocrKey** prop or you can passURL address via **ocrUrl** prop in case of own backend realisation of Google Vision API call.  

#### Usage:

```typescript jsx
import React from 'react';
import { AssetScanner } from '@cognite/gearbox';

function ExampleComponent(props) {
  const onUnauthorized = (error: any): void => {};
  return (
    <AssetScanner
      onUnauthorized={onUnauthorized}
      ocrKey={'YOUR_GOOGLE_VISION_KEY'}
    />
    
  );
  
}
```

#### Available props:

##### Required:

> _No required props_

Reviewer: Really no one?

Author: users can use scanner without providing any props, but they might not get expected behaviour without providing some of optional params. In storybook they can find description of all props with some examples

##### Optionals:

| Property             | Description                                                                | Type                             | Default                                          |
| -------------------- | -------------------------------------------------------------------------- | -------------------------------- | ------------------------------------------------ |
| `ocrUrl`             | URL of detect service (OCR) API                                            | `string`                         | https://vision.googleapis.com/v1/images:annotate |
| `ocrKey`             | Key property for OCR service                                               | `{ [name: string]: string }`     |                                                  |

> | `customNotification` | Function which is react on provided type of notification                   | `(type: ASNotifyTypes) => any`   |                                                  |

Reviewer: ```suggestion
| `customNotification` | Callback function to react on provided type of notification                   | `(type: ASNotifyTypes) => any`   |                                                  |
```

> | `onStringRecognize`  | Callback which is triggering when strings was recognised on image          | `(strings: string[]) => void;`   |                                                  |

Reviewer: ```suggestion
| `onStringRecognize`  | Callback which is triggering when strings was recognised in image          | `(strings: string[]) => void;`   |                                                  |
```
